pkg/awsutils/awsutils.go

```diff
@@ -224,8 +225,47 @@ func prometheusRegister() {
 	}
 }
 
+//StringSet is a set of strings
+type StringSet struct {
+	sync.RWMutex
+	data 	sets.String
+}
+
+func (ss *StringSet) AWSStrings() []*string {
+	ss.RLock()
+	defer ss.RUnlock()
+	var dataSlice []*string
+	for key, _ := range ss.data {
+		dataSlice = append(dataSlice, aws.String(key))
+	}
+	return dataSlice
+}
+
+func (ss *StringSet) Set(items []string) {
+	ss.Lock()
+	defer ss.Unlock()
+	ss.data = sets.NewString(items...)
+}
+
+func (ss *StringSet) IsEmpty() bool {
+	if ss.data.Len() == 0 {
```

Comment:
nit.. this assumes a non-nil `data` field. Plus, you could simplify this method to:

```go
return ss.data != nil && ss.data.Len() == 0
```

Reply:
Fixed in #1056